app/ante/evm/06_account_verification_test.go

```diff
@@ -0,0 +1,139 @@
+package evm_test
+
+import (
+	"fmt"
+	"math/big"
+
+	"cosmossdk.io/math"
+	errortypes "github.com/cosmos/cosmos-sdk/types/errors"
+	"github.com/evmos/evmos/v16/app/ante/evm"
+	"github.com/evmos/evmos/v16/testutil/integration/evmos/factory"
+	"github.com/evmos/evmos/v16/testutil/integration/evmos/grpc"
+	testkeyring "github.com/evmos/evmos/v16/testutil/integration/evmos/keyring"
+	"github.com/evmos/evmos/v16/testutil/integration/evmos/network"
+	"github.com/evmos/evmos/v16/x/evm/statedb"
+	evmtypes "github.com/evmos/evmos/v16/x/evm/types"
+)
+
+const SENDER_KEYRING_KEY = 1
```

Comment:
```suggestion
const SenderKeyringKey = 1
```

Reply:
sorry had done this changes but had forgot to push